docs/relational-databases/backup-restore/sql-server-managed-backup-to-microsoft-azure.md

```diff
@@ -133,6 +133,8 @@ The following support limitations and considerations are specific to [!INCLUDE [
 
 - Backups of databases in an availability group are [copy-only backups](copy-only-backups-sql-server.md).
 
+- Backups of databases on secondary replica's in an availabiity group need to be readable in order to stripe to multiple files. 
```

Comment:
```suggestion
- Backups of databases on secondary replicas in an availability group must be readable in order to stripe to multiple files. 
```

Reply:
already pulled into my own PR and updated :) <3